CONTRIBUTING.md

```diff
@@ -130,6 +130,12 @@ To make it easier for reviewers to review your PR, consider the following:
    cannot be fixed automatically, an error message will be displayed so you can address the issue.
 3. Follow [git commit](https://chris.beams.io/posts/git-commit/) guidelines.
 4. Follow [logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md) guidelines.
+5. Check below table for some general words and abbreviations you might use.
+
+| Word/Phrase | Word/Abbr | Notes |
+|---|---|---|
+| IP Security | IPsec | refer to [rfc 6071](https://datatracker.ietf.org/doc/html/rfc6071) |
+| Namespace, Pod, Service... | Namespace, Pod, Service... | Any Kubernetes object in log/comment should start with upper case |
```

Comment:
I don't know if a table really makes sense here since not everything is going to be about abbreviations (the second rule for example). Maybe we should have a separate section `Conventions for Writing Documentation` in this doc and simply use a bullet point list for now. You can add a link to the new section here.

Reply:
done, Could you help to take a look again? thanks.